### PR TITLE
For examples/meteor: update README, minor fixes.

### PR DESCRIPTION
**Pull request by jeffmendoza**
Make readme more generic, fix editorial issues.
Remove resource requirements from pod definitions.

### CODE REVIEW

File: examples/meteor/v1beta1/meteor-controller.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (fabioy) [inline examples/meteor/v1beta1/meteor-controller.json]: Are you sure you want to add v1beta1 versions of these files? They may have a rather short lifecycle.

**Comment by fabioy** [comment]
Thanks for the update. Very detailed explanation! Just a minor comment.

File: examples/meteor/v1beta1/meteor-controller.json (hunk 1 above)
Comment (jeffmendoza) [inline examples/meteor/v1beta1/meteor-controller.json]: Ah, yes,. I thought the guidance was to copy the old files into a subdir for now. If they're not needed, I'll remove them.

Comment (fabioy) [inline examples/meteor/v1beta1/meteor-controller.json]: Given that v1beta1 is going away soon, I'd rather not clutter the depot with them. Please remove them.

**Comment by jeffmendoza** [comment]
@fabioy removed v1beta1 definitions.

**Comment by fabioy** [comment]
LGTM. Will merge on green. Thank you.

**Pull request merged**